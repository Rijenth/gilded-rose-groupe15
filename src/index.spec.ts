import { GildedRose, Item } from ".";
import runGoldenMaster from "jest-golden-master";

describe("Gilded Rose", function () {
    it("should foo", function () {
        runGoldenMaster(async () => {
            const gildedRose = new GildedRose([new Item("foo", 0, 0)]);

            gildedRose.updateQuality();
        });
    });
});