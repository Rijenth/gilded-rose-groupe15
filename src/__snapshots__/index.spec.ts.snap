// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gilded Rose Test Aged Brie 1`] = `
[
  "Mise à jour du produit",
  "Le produit est soit Aged Brie, soit Backstage passes",
  "La qualité du produit est inférieure à 50, augmentation de la qualité",
  "Le produit n'est pas Sulfuras, réduction de SellIn",
  "Produit mis à jour",
]
`;

exports[`Gilded Rose Test Backstage passes 1`] = `
[
  "Mise à jour du produit",
  "Le produit est soit Aged Brie, soit Backstage passes",
  "La qualité du produit est inférieure à 50, augmentation de la qualité",
  "Le produit est Backstage passes",
  "SellIn est inférieur à 11",
  "La qualité est inférieure à 50, augmentation de la qualité",
  "Le produit n'est pas Sulfuras, réduction de SellIn",
  "Produit mis à jour",
]
`;

exports[`Gilded Rose Test Conjured items 1`] = `
[
  "Mise à jour du produit",
  "Le produit n'est ni Aged Brie ni Backstage passes",
  "La qualité du produit est supérieure à 0",
  "Le produit n'est pas Sulfuras, réduction de la qualité",
  "Le produit n'est pas Sulfuras, réduction de SellIn",
  "Produit mis à jour",
]
`;

exports[`Gilded Rose Test Sulfuras 1`] = `
[
  "Mise à jour du produit",
  "Le produit n'est ni Aged Brie ni Backstage passes",
  "La qualité du produit est supérieure à 0",
  "Produit mis à jour",
]
`;

exports[`Gilded Rose Test normal item 1`] = `
[
  "Mise à jour du produit",
  "Le produit n'est ni Aged Brie ni Backstage passes",
  "La qualité du produit est supérieure à 0",
  "Le produit n'est pas Sulfuras, réduction de la qualité",
  "Le produit n'est pas Sulfuras, réduction de SellIn",
  "Produit mis à jour",
]
`;
